CMakeLists.txt

```diff
@@ -35,6 +35,7 @@ option(protobuf_BUILD_LIBUPB "Build libupb" ON)
 option(protobuf_DISABLE_RTTI "Remove runtime type information in the binaries" OFF)
 option(protobuf_TEST_XML_OUTDIR "Output directory for XML logs from tests." "")
 option(protobuf_ALLOW_CCACHE "Adjust build flags to allow for ccache support." OFF)
+option(protobuf_USE_UNITY_BUILD "Enable Unity (Jumbo) build for" ON)
```

Comment:
Our .NET support policy states that we explicitly don't support Unity (https://opensource.google/documentation/policies/dotnet-support) and thus won't have tests, etc for these environments. We may be able to accept a contribution here anyways, but note that support would be best-effort / community-contributed, and this should at least be noted in a comment here accordingly.

Reply:
Sorry, I think there is a confusion. I'm not talking about building for the Unity game engine.

This is about having unity-build: [Unity-Jumbo Build](https://en.wikipedia.org/wiki/Unity_build) A build in which all the cpp files are merged together into a big translation unit.

Reply:
Ah that makes more sense and seems more feasible then -- thanks for clarifying!